Homework check #2

Lewkurs opened the pull request:
Implemented JSON, updated to use Remote MYSQL Server, Refined Menu for project & Parsers

Errors Still Found In:
ReviewsDAOImpl - Lines 71 & 103
DestinationBookingsDAOImpl - Lines 57 & 81 & 102
HotelBookingsDAOImpl - Lines 49 - 51, 67-69,  83 & 84,

Lewkurs committed:
Add DAOImpl classes and xml/xsd files for all classes - refactored Services and DAO classes

Lewkurs committed:
Add DAOImpl classes and xml/xsd files for all classes - refactored Services and DAO classes

Lewkurs committed:
JSON, Remote MYSQL Server, Refine Menu for project & Parsers

Errors Still Found In:
ReviewsDAOImpl - Lines 71 & 103
DestinationBookingsDAOImpl - Lines 57 & 81 & 102
HotelBookingsDAOImpl - Lines 49 - 51, 67-69,  83 & 84,

Old-Nem11 commented:
Lots of changes made, looks good so far!

aostapenko92 commented:
-DOM Parser looks good. 
-I don't see JAXB annotations in the project, it needs to be done before Tuesday. 
-We still need to work on the business logic of the application. I want to see calculations and work with other objects and data from the database. 
-Please add .idea and .gitattributes to .gitignore file and fix the mistakes mentioned above.

Dnelson16 reviewed (commented):
JSON parser looks good.
I don't see and JAXB annotations.
Still see some System.out.Println.
Overall, the structure you have is good, I would add some more business logic to the project.

Lewkurs committed:
Added JAXB Annotations

Lewkurs committed:
Added MyBatis Mappers and implemented some logic in MyBatisRunner - Fixed bugs in my DAOImpl classes and added more logic to Menu class that allows user to transfer between Parsers or Travel Agency

Having some issues with my file paths - it keeps saying it cannot find my Mappers.xml when I run my code but the path seems correct.
